internal/k8s/controller.go

```diff
@@ -743,6 +746,12 @@ func (lbc *LoadBalancerController) sync(task task) {
 	case transportserver:
 		lbc.syncTransportServer(task)
 	}
+
+	if lbc.firstRun && lbc.syncQueue.Len() == 0 {
```

Comment:
I wonder if we need to have the `firstRun` field at all. What if we had just this:
```go
	if !lbc.isNginxReady && lbc.syncQueue.Len() == 0 {
		lbc.isNginxReady = true
		glog.V(3).Infof("NGINX is ready")
	}
```

Reply:
I think we need it to know if the flag is enabled or not.

Reply:
@lucacome 
is there any harm if we do that check regardless of whether `firstRun` enabled or not?